README.adoc

```diff
@@ -4062,7 +4062,7 @@ a.fdiv(b)
 
 When using exponential notation for numbers, prefer using the normalized scientific notation, which uses a mantissa between 1 (inclusive) and 10 (exclusive). Omit the exponent altogether if it is zero.
 
-The goal is to avoid confusion between powers of ten and exponential notation, as one quickly reading `10e7` could think it's `10**7` when it's actually `10**8`.
```

Comment:
I think a simpler fix would be to use the passthrough notation (`+ .. +`). See https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#formatted-text

Reply:
Btw, AsciiDoc actually has superscripts, so I guess that'd be even better - `10^7^`.